### PR TITLE
Fix the bug report issue template

### LINKED ISSUE #1426: fix the bug report issue template
There are few issues with the bug report issue template.

The version selector doesn't have all the version, we might just remove this and ask the user for output of `buildtest --version` instead. Keeping this up to date for every version can be very tedious and not worth it.  We could even ask for the HEAD commit `git log -1` to see what commit they are on.

Screen Shot 2023-03-17 at 9 39 43 AM

We can ask user to provide information for `buildtest info` and `buildtest debugreport` to help us troubleshoot the issue. They can be optional fields.

It could be helpful to have section `How to reproduce error` which can be used for user to specify instructions on reproducing the error messages. The `Bug description` message can be used to describe the message.

### PR DESCRIPTION
**Pull request by prathmesh4321**
Hi @shahzebsiddiqui . I have added to the bug report issue template. Please review the PR.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -10,56 +10,53 @@ body:
...
         ```
+    validations:
+      required: true
 
+  - type: textarea
+    id: reproduce-error
+    attributes:
+      label: Steps to reproduce the error
+      description: |
+        Please provide a list of steps to reproduce the error inorder for us to troubleshoot your issue.
```
Comment (shahzebsiddiqui) [inline .github/ISSUE_TEMPLATE/bug.yml]: `inorder` -> `in-order`

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -10,56 +10,53 @@ body:
...
-        - v0.9.3
-        - v0.9.2
-        - v0.9.1
-        - v0.9.0
-        - v0.8.1
-        - v0.8.0
-        - other
+      label: Version and HEAD commit
+      description: |
+        Please copy and paste output of the commands "buildtest --version" and "git log -1" in order for us to know the
```
Comment (shahzebsiddiqui) [inline .github/ISSUE_TEMPLATE/bug.yml]: i think you might be able to format the text "buildtest --version" and "git log -1" with single ticks. Take a look at rendered text https://github.com/buildtesters/buildtest/blob/bug_report_issue_template/.github/ISSUE_TEMPLATE/bug.yml when you make change

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -10,56 +10,53 @@ body:
...
     validations:
       required: true
 
   - type: textarea
     id: logs
     attributes:
       label: Relevant log output
-      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      description: |
+        Please copy and paste any relevant log output. Please provide the output of the commands "buildtest info" and "buildtest debugreport"
```
Comment (shahzebsiddiqui) [inline .github/ISSUE_TEMPLATE/bug.yml]: backticks around `buildtest info` and `buildtest debugreport`

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 4]
```diff
@@ -10,56 +10,53 @@ body:
...
+        in order for us to know about the buildtest setup and configuration.
```
Comment (shahzebsiddiqui) [inline .github/ISSUE_TEMPLATE/bug.yml]: lets keep this short remove `in order for us to know about the buildtest setup and configuration.`

Comment (prathmesh4321) [reply .github/ISSUE_TEMPLATE/bug.yml]: @shahzebsiddiqui Okay, I have made the changes.

**Review by shahzebsiddiqui — APPROVED** [review]
looks good. Thanks @prathmesh4321

**Pull request merged**